File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/Staged.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class Staged {
    @JsonProperty("CommissionedDateTime")
    public String commissionedDateTime;
    @JsonProperty("DetachedDateTime")
    public String detachedDateTime;
    @JsonProperty("FacilityId")
    public Integer facilityId;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("IsArchived")
    public Boolean isArchived;
    @JsonProperty("IsStaged")
    public Boolean isStaged;
    @JsonProperty("IsUsed")
    public Boolean isUsed;
    @JsonProperty("Label")
    public String label;
    @JsonProperty("LastModified")
    public String lastModified;
    @JsonProperty("MaxGroupSize")
    public Integer maxGroupSize;
    @JsonProperty("ProductLabel")
    public String productLabel;
    @JsonProperty("QrCount")
    public Integer qrCount;
    @JsonProperty("StatusName")
    public String statusName;
    @JsonProperty("TagInventoryTypeName")
    public String tagInventoryTypeName;
    @JsonProperty("TagTypeId")
    public Integer tagTypeId;
    @JsonProperty("TagTypeName")
    public String tagTypeName;
    @JsonProperty("UsedDateTime")
    public String usedDateTime;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/Staged.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class Staged(
    @JsonProperty("CommissionedDateTime")
    val commissionedDateTime: String? = null,
    @JsonProperty("DetachedDateTime")
    val detachedDateTime: String? = null,
    @JsonProperty("FacilityId")
    val facilityId: Int? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("IsArchived")
    val isArchived: Boolean? = null,
    @JsonProperty("IsStaged")
    val isStaged: Boolean? = null,
    @JsonProperty("IsUsed")
    val isUsed: Boolean? = null,
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("LastModified")
    val lastModified: String? = null,
    @JsonProperty("MaxGroupSize")
    val maxGroupSize: Int? = null,
    @JsonProperty("ProductLabel")
    val productLabel: String? = null,
    @JsonProperty("QrCount")
    val qrCount: Int? = null,
    @JsonProperty("StatusName")
    val statusName: String? = null,
    @JsonProperty("TagInventoryTypeName")
    val tagInventoryTypeName: String? = null,
    @JsonProperty("TagTypeId")
    val tagTypeId: Int? = null,
    @JsonProperty("TagTypeName")
    val tagTypeName: String? = null,
    @JsonProperty("UsedDateTime")
    val usedDateTime: String? = null
)
